packages/subgraph/src/v1-analytics/mapping.ts

```diff
@@ -257,6 +297,7 @@ export function handleTransactionFulfilled(event: TransactionFulfilled): void {
       hourlyMetric.liquidity = assetBalance.amount;
     }
 
+    hourlyMetric.sendingTxCount = hourlyMetric.receivingTxCount.plus(BigInt.fromI32(1));
```

Comment:
why receivintTxCount + 1 for sendingTxCount???

I think it should be like this
`hourlyMetric.sendingTxCount = hourlyMetric.sendingTxCount .plus(BigInt.fromI32(1));`

Reply:
ahh yep!